README.md

```diff
@@ -1,8 +1,111 @@
-# eventing-auth-manager
-// TODO(user): Add simple overview of use/purpose
+# Eventing Auth Manager
+The eventing-auth-manager is a central component that is deployed on the [Kyma control-plane](https://github.com/kyma-project/control-plane). The component manages applications in the 
+SAP Cloud Identity Services - Identity Authentication (IAS) by creating and deleting them based on creation or deletion of a managed Kyma Runtime.
+
+### Table of contents
+* [Description](#description)
+* [EventingAuth CR](#eventingauth-cr)
+* [eventing-webhook-auth secret](#eventing-webhook-auth-secret)
+* [Name reference between resources](#name-reference-between-resources)
+* [Design decisions](#design-decisions)
+* [Generating the SAP Cloud Identity Services API client](#generating-the-sap-cloud-identity-services-api-client)
+* [Getting Started](#getting-started)
+* [Contributing](#contributing)
+
 
 ## Description
-// TODO(user): An in-depth paragraph about your project and overview of use
+The eventing-auth-manager manages the credentials for IAS applications used by webhooks in the `eventing` component.  
+When a new managed Kyma Runtime is provisioned the component creates a new [OIDC](https://openid.net/connect/) IAS application for this runtime and stores the credentials in a secret on the new runtime.
+
+![eventing-auth-manager-overview](./doc/overview.svg)
+
+A Kyma CR is created for each runtime. The Eventing Auth Manager watches the creation and deletion of Kyma CRs. On the creation of a Kyma CR, the Eventing Auth Manager creates an EventingAuth CR. 
+The reconciliation of the EventingAuth CR will create an application in IAS using the [Application Directory REST API](https://api.sap.com/api/SCI_Application_Directory/) and the secret with the credentials on the managed runtime.
+When the Kyma CR is deleted, the controller deletes the EventingAuth CR. On the deletion of the EventingAuth CR the Eventing Auth Manager deletes the application in IAS and the secret on the runtime.
+
+![controller-flow](./doc/controller-flow.svg)
```

Comment:
Nice dagramm! 
I have a couple of comments here:
* Can we rename the step `Create client secret for IAS application` to `Create IAS Application client secret`? I am asking this because when I read the current one I always got it like K8s secret. (nitpick)
* Can we rename the step `Create secret on managed runtime` to `Create K8s secret on managed runtime`. Otherwise, it was not clear from the first read what kind of secret it is.
* Can we merge `Runtime deletion` and `Reconcile EventingAuth deletion` to ? The reason is this makes is really easy to understand from the first sight.

Reply:
Thanks, those are very good points